src/sbt-test/sbt-scalafix/cross-build-advanced/.scalafix-2.conf

```diff
@@ -0,0 +1,3 @@
+rules = [DisableSyntax, OrganizeImports, ExplicitResultTypes]
```

Comment:
DisableSyntax seems unexercised, but it doesn't hurt to keep a syntactic rule here I guess